group.go

```diff
@@ -35,6 +35,9 @@ type Scalar interface {
 	// SetInt64 sets the receiver to a small integer value.
 	SetInt64(v int64) Scalar
 
+	// SetIntString sets the receiver to a string encoded integer value.
+	SetIntString(v string) (Scalar, error)
```

Comment:
I guess my main worry with making the Scalar interface any bigger is that it'll break all implementations using custom scalars.

That being said we're en route to V4 so we _could_ do it... Opinions @pierluca  ?

Reply:
I share your concern, @AnomalRoil . Could you elaborate a little bit more about the use case and why this needs to appear in the Scalar interface, @pmcampones ?

Reply:
It says "string encoded integer value", which definitely looks like it should be done using `SetInt64`. Other possibilities I see:

- `SetBigInt` which could use more than 64 bits
- `SetBytes` which already exists :)

So I suppose this should be something like `SetInt64(strconv.Atoi())` or use the `SetBytes` for bigger integers.

Reply:
Maybe SetBigInt would be a better name for the method.
In the comment thread bellow we discussed using only SetBytes instead of the SetIntString.
I find using SetIntString a bit more convenient and less error prone than using SetBytes.